environment.yml

```diff
@@ -9,9 +9,9 @@ dependencies:
   - mpi4py>=3.0.0
   - numpy>=1.15
   - pip
+  - pre-commit
```

Comment:
I'm a little weary about this choice. I have pulled `pre-commit` from conda before and gotten a broken build. But I'm not one to dictate where to get packages from. Maybe just make a mental note in case things get weird one day.

Reply:
weary or wary? ;) I guess I just prefer to get everything I can from conda and this is what I've been doing on all my local environments. But I don't want to push people to something that's likely to be broken. I could add a comment within the yaml if you think that's a good idea.

Reply:
Clearly I'm too weary to be wary about the differences between those words. The dangers of not consuming coffee in the morning. If it is working now, maybe I just got a single bad build. I think if this works, let's just stick with it.